pkgs/os-specific/linux/nvidia-x11/builder.sh

```diff
@@ -66,11 +66,19 @@ installPhase() {
             sed "s#__NV_VK_ICD__#$i/lib/libGLX_nvidia.so#" nvidia_icd.json.template > nvidia_icd.json
         fi
         if [ -e nvidia_icd.json ]; then
-            install -Dm644 nvidia_icd.json $i/share/vulkan/icd.d/nvidia.json
+            sed -E "s#(libGLX_nvidia)#$i/lib/\\1#" nvidia_icd.json > nvidia_icd.json.fixed
```

Comment:
It looks like an issue after recent rebase (due PR #68024). I think with old drivers it will create nvidia_icd.json from template with full path to libGLX_nvidia and in second pass (due "-e nvidia_icd.json") it will process that file once again leading to "/nix/hash/lib/nix/hash/lib/libGLX_nvidia.so". We need `elif` here and also add install step to `nvidia_icd.json.template`

Reply:
Also I don't understand why in #68024 in case of 435 driver they don't put full path to libGLX_nvidia in `nvidia_icd.json`